src/utils/api.js

```diff
@@ -0,0 +1,9 @@
+export const fetchApiData = async (endpoint) => {
```

Comment:
This is the fetch method with the interpolated string designating project or palette endpoint.